app/apps/server/orchestrator.js

```diff
@@ -72,6 +72,24 @@ class AppServerOrchestrator {
 		return this._manager;
 	}
 
+	getEnabledGames() {
+		const apps = this._manager.get().map((prl) => {
+			const info = prl.getInfo();
+			info.status = prl.getStatus();
+			return info;
+		});
+		const activatedApps = apps.filter(
+			({ status }) => status === 'manually_enabled'
+		).map(({ id }) => id);
+		const { externalComponentManager: { providedComponents } } = this._manager;
+		Array.from(providedComponents.keys()).forEach((id) => {
```

Comment:
@lolimay this check is not needed - the Apps-Engine will only provide components when an app is properly enabled

Reply:
ok, thank you!